### PR TITLE
fix: opening filter menu in data browser leads to blank page

### PR DESCRIPTION
**Pull request by TomKremer**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).

### Issue Description

Related issue: #1915

### TODOs before merging

- [x] A changelog entry is created automatically using the pull request title (do not manually add a changelog entry)

### CODE REVIEW

**Review by andreafalzetti — APPROVED** [review]

File: src/components/Filter/Filter.react.js [hunk 1]
```diff
@@ -61,7 +61,8 @@ let Filter = ({ schema, filters, renderRow, onChange, blacklist, className }) =>
         }
 
         // Get the column preference of the current class.
-        const currentColumnPreference = currentApp.columnPreference[className];
+        const currentColumnPreference = currentApp.columnPreference ? currentApp.columnPreference[className]
+        : null;
```
Comment (mtrezza) [inline src/components/Filter/Filter.react.js]: Is this linted? I'd assume `: null` either be appended to the previous line or would be indented in its current line.

Comment (TomKremer) [reply src/components/Filter/Filter.react.js]: linting did not work for me. Fixed it manually.

**Review by mtrezza — APPROVED** [review]
Works fine, thanks for the fix!

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0-alpha.5](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.0-alpha.5)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0-beta.1](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.0)

**Pull request merged**